src/ImageKit/ImageKit.php

```diff
@@ -174,10 +218,24 @@ public function uploadFiles(array $options)
      * @param array $parameters
      * @return Response
      */
-    public function listFiles(array $parameters = [])
+    public function listFiles($parameters = null)
     {
+        if($parameters){
+            if(!is_array($parameters)){
+                return Response::respond(true, ((object)ErrorMessages::$LIST_FILES_OPTIONS_NON_ARRAY));
+            }
+        }
+
+        if (isset($parameters['tags']) && is_array($parameters['tags'])) {
+            $parameters['tags'] = implode(',', $parameters['tags']);
```

Comment:
There could be a few other array properties that need to be converted too comma-separated string 
https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload
E.g. `responseFields`

Reply:
removed this conversion becuase in `listFiles` it is not required, added the conversion code to `upload API`